node/src/types/block.rs

```diff
@@ -765,7 +765,7 @@ impl BlockHeader {
     /// Returns true if block is Genesis' child.
     /// Genesis child block is from era 0 and height 0.
     pub(crate) fn is_genesis_child(&self) -> bool {
-        self.era_id() == EraId(0) && self.height() == 0
+        self.era_id() == EraId::from(0) && self.height() == 0
```

Comment:
I'd always use `is_genesis` instead of comparing with `0`.
```suggestion
        self.era_id().is_genesis() && self.height() == 0
```